docs/projects/tad/reporting-standard/latest.md

```diff
@@ -73,10 +73,12 @@ A `system_card` contains the following information.
     4. `author` (OPTIONAL, string). Name of person that initiated the transformations.
 
 3. `name` (OPTIONAL, string). Name used to describe the system.
-4. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+4. `instrument_urns` (OPTIONAL, list[str]). List of Uniform Resource Names of instruments from
```

Comment:
Why instrument_urns and not just instruments?

Reply:
Processed your suggestion.